### PR TITLE
Target .NET 5

### PR DESCRIPTION
**Pull request by UselessToucan**
- [x] Needs testing on mobile platforms (iOS in particular)

### CODE REVIEW

File: osu.Game/osu.Game.csproj [hunk 1]
```diff
@@ -18,11 +18,13 @@
...
   </ItemGroup>
   <ItemGroup Label="Package References">
+    <!-- WorkaroundWorkaround for a System.IO.FileNotFoundException on startup. Required because of old EF Core version. -->
+    <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" Version="2.2.0" />
     <PackageReference Include="Dapper" Version="2.0.35" />
     <PackageReference Include="DiffPlex" Version="1.6.3" />
     <PackageReference Include="Humanizer" Version="2.8.26" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="3.1.9" />
-    <PackageReference Include="Microsoft.AspNetCore.SignalR.Protocols.NewtonsoftJson" Version="3.1.9" />
+    <PackageReference Include="Microsoft.AspNetCore.SignalR.Client" Version="5.0.0" />
```
Comment (peppy) [inline osu.Game/osu.Game.csproj:26]: will need re-testing of mobile platforms as we have them pinned to a much earlier version for compatibility reasons.

**Comment by huoyaoyuan** [comment]
I'd suggest to separate this into multiple parts. Updating SDK and entry executable to .NET 5 can unblock SDK feature opt-in.

File: osu.Game/osu.Game.csproj (hunk 1 above)
Comment (peppy) [reply osu.Game/osu.Game.csproj]: These changes were reverted, no longer an issue.

**Review by bdach — APPROVED** [review]

**Comment by bdach** [comment]
Tested android and windows, from basic checks it seems to work fine. I suppose time to get this in and see if anything falls out.

**Review by peppy — APPROVED** [review]

**Pull request merged**